src/pool/srv_pool.c

```diff
@@ -105,7 +105,8 @@ pool_hdl_size(struct pool_svc *svc)
 	if (svc->ps_global_version != 0)
 		return sizeof(struct pool_hdl);
 	else
-		return sizeof(struct pool_hdl) - sizeof(((struct pool_hdl *)0)->ph_machine);
+		return sizeof(((struct pool_hdl *)0)->ph_flags) +
```

Comment:
How is this different than sizeof(struct pool_hdl) ?   In the old format, these are the only two fields in the struct

Reply:
> How is this different than sizeof(struct pool_hdl) ? In the old format, these are the only two fields in the struct

@jolivier23 do you mean, if this code runs in 2.0 ? But it is assumed it will not here. Or if it will, do I need to also check against DS_POOL_GLOBAL_VERSION ?

Reply:
ah, right. nevermind.